File: model/src/main/java/com/zj/model/city/TopGeoBean.java
package com.zj.model.city;

import com.zj.model.Refer;

import java.util.List;

public class TopGeoBean {
    private String code;
    private Refer refer;
    private List<GeoBean.LocationBean> topCityList;

    public String getCode() {
        return code;
    }

    public Refer getRefer() {
        return refer;
    }

    public List<GeoBean.LocationBean> getTopCityList() {
        return topCityList;
    }
}

File: model/src/main/java/com/zj/model/city/TopGeoBean.kt
package com.zj.model.city

import com.zj.model.Refer
import com.zj.model.city.GeoBean.LocationBean

data class TopGeoBean(
    val code: String? = null,
    val refer: Refer? = null,
    val topCityList: List<LocationBean>? = null
)